practice-two/src/js/script.js

```diff
@@ -1 +1,64 @@
+const getInfo = () => {
+  //Query to id
+  const fullName = document.getElementById('fullName');
+  const email = document.getElementById('email');
+  const salary = document.getElementById('salary');
+  const city = document.getElementById('city');
+  //From the initial id return the parent element then query the child class
+  const errMsgFullName = fullName.parentElement.querySelector('.warn-msg');
+  const errMsgEmail = email.parentElement.querySelector('.warn-msg');
+  const errMsgSalary = salary.parentElement.querySelector('.warn-msg');
+  const errMsgCity = city.parentElement.querySelector('.warn-msg');
+  //Check the input condition
+  const regalphabet = /^[a-zA-Z]+ [a-zA-Z]+$/;
```

Comment:
Instead of local variable, use it as Global variable, because it can be used everywhere